nautobot/ipam/models.py

```diff
@@ -983,6 +983,8 @@ class VLANGroup(OrganizationalModel):
         null=True,
     )
     description = models.CharField(max_length=200, blank=True)
+    min_vid = models.PositiveIntegerField(verbose_name="Minimum VLAN ID", default=1)
+    max_vid = models.PositiveIntegerField(verbose_name="Maximum VLAN ID", default=4094)
```

Comment:
1) Can we use the `VLAN_VID_MIN` and `VLAN_VID_MAX` constants as defaults here instead of literal 1 and 4094?
2) I'd expect you need to add a `clean()` method to this model now that asserts that a) `min_vid <= max_vid` b) `min_vid >= VLAN_VID_MIN` c) `max_vid <= VLAN_VID_MAX` so as to protect against setting invalid ranges.